### PR TITLE
fix: mkdocs community/features-templating

### PR DESCRIPTION
**Pull request by daoane**
Updated/fixed markdown documentation for community/features-templating.

### CODE REVIEW

**Comment by aaime** [comment]
Do you know if the documentation here has anything original compared to the official docs at:
https://docs.geoserver.org/latest/en/user/community/features-templating/index.html

I'd rather keep documentation in one place, the place above is where it should be.

**Comment by jodygarnett** [comment]
Please we are just updating the rst files right now until the conversion is completed

File: doc/en/user/docs/community/features-templating/directives.md [hunk 1]
```diff
@@ -12,49 +12,31 @@ The following constitutes a summary of all the template directives and it is mea
...
-
-  allows a template to extend another template                         \$merge                              specify the `$merge` directive as an attribute name containing the path to the extended template (:code: ``"\$merged":"base_template.json"``).
-
-  allows null values to be encoded. default is not encoded.            \${property}! or \$\${expression}!   ! at the end of a property interpolation or cql directive (`"attribute":"${property}!"` or `"attribute":"$${expression}!"`).
-  -------------------------------------------------------------------- ------------------------------------ --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+| **Usage** | **Syntax** | **Description** |
+|-|-|-|
+|property interpolation | \${property} | specify it as an attribute value (`"json_attribute":"${property}"`) |
+| cql evaluation | \$\${cql} | specify it as an element value (`"json_attribute":"$${cql}"`) |
```
Comment (jodygarnett) [inline doc/en/user/docs/community/features-templating/directives.md:18]: I wish I could tell what you were changing there, checking the rst conversion I am finding that these were intended to be code examples and are getting \\$ shown in the output.

**Comment by jodygarnett** [comment]
Thanks for pointing out the problems; I was able to start post-processing grid-tables coming out of pandoc and support more tables then previously.

**Comment by aaime** [comment]
@jodygarnett mind these docs were slapped into the module root directory directly as markdown, before we could convince the author to document using RST :rofl:

**Comment by daoane** [comment]
Unfortunately, I wasn't able to participate in your discussion before the pull request was closed.

@jodygarnett : 
What I did was to format the mkdocs as similar to the original rst documentation as possible. Side-by-side, checking on two screens. Not all styling is applied in forks. Thus, I can't tell to 100 % that things are alright. But this is mainly about checking the format of warnings, etc.

I was waiting for this pull request to be merged, to be sure that everything is as it should. Apparently, it is not, but I can't tell what exactly the problem is. Please let me know whether I should continue with reviewing documentation and if so, what to do differently. Alternatively, I can cancel what I signed up for in the documentation spreadsheet and revert these lines to the state before my attempted contribution.

@aaime : I am not sure what you mean with "mind these docs were slapped into the module root directory directly as markdown" - I forked, edited existing documents in the mkdocs branch and then created a pull request to the mkdocs branch. No new files involved at all.

**Comment by daoane** [comment]
Ok, then I will read @jodygarnett 's request for help and instructions again, as I apparently have completely misunderstood the entire undertaking, including how problems should fixed and after reporting in the mkdocs spreadsheet.

**Pull request closed without merging**